src/native-automation/request-pipeline/index.ts

```diff
@@ -154,12 +154,13 @@ export default class NativeAutomationRequestPipeline extends NativeAutomationApi
             await this._resourceInjector.processNonProxiedContent(fulfillInfo, this._client, sessionId);
         else {
             const userScripts = await this._getUserScripts(event);
+            const contentType = getHeaderValue(event.responseHeaders, 'content-type');
```

Comment:
please check if we can use constants from `\src\utils\http-headers.ts` instead of string `content-type`